docs/rafter/05-01-rafter.md

```diff
@@ -0,0 +1,47 @@
+| **controller-manager.minio.resources.requests.memory** | Defines requests for memory resources. | `32Mi` |
+| **controller-manager.minio.resources.requests.cpu** |  Defines requests for CPU resources. | `10m` |
+| **controller-manager.minio.resources.limits.memory** |  Defines limits for memory resources. | `128Mi` |
+| **controller-manager.minio.resources.limits.cpu** | Defines limits for CPU resources. | `100m` |
+| **controller-manager.deployment.replicas** | Defines the number of service replicas. | `1` |
+| **controller-manager.pod.resources.limits.cpu** |  Defines limits for CPU resources. | `150m` |
+| **controller-manager.pod.resources.limits.memory** | Defines limits for memory resources. | `128Mi` |
+| **controller-manager.pod.resources.requests.cpu** | Defines requests for CPU resources. | `10m` |
+| **controller-manager.pod.resources.requests.memory** | Defines requests for memory resources. | `32Mi` |
+| **controller-manager.envs.clusterAssetGroup.relistInterval** | Determines time intervals in which the Rafter Controller Manager verifies the ClusterAssetGroup for changes. | `5m` |
+| **controller-manager.envs.assetGroup.relistInterval** | Determines time intervals in which the Rafter Controller Manager verifies the AssetGroup for changes. | `5m` |
+| **controller-manager.envs.clusterBucket.region** | Specifies the regional location of the ClusterBucket in a given cloud storage. Use one of [these](https://github.com/kyma-project/kyma/blob/master/resources/cluster-essentials/templates/bucket.crd.yaml#L52) regions. | `us-east-1` |
+| **controller-manager.envs.bucket.region.value** | Specifies the regional location of the bucket in a given cloud storage. Use one of [these](https://github.com/kyma-project/kyma/blob/master/resources/cluster-essentials/templates/bucket.crd.yaml#L52) regions. | `us-east-1` |
+| **controller-manager.envs.clusterBucket.maxConcurrentReconciles** | Defines the maximum number of cluster bucket concurrent reconciles which will run. | `1` |
+| **controller-manager.envs.bucket.maxConcurrentReconciles** | Defines the maximum number of bucket concurrent reconciles which will run. | `1` |
+| **controller-manager.envs.clusterAsset.maxConcurrentReconciles** | Defines the maximum number of cluster asset concurrent reconciles which will run. | `1` |
+| **controller-manager.envs.asset.maxConcurrentReconciles** | Defines the maximum number of asset concurrent reconciles which will run. | `1` |
+| **controller-manager.envs.store.secretKey** | Defines the secret key. Add the parameter to set your own **secretkey** credentials. | By default, **secretKey** is automatically generated. |
+| **controller-manager.envs.store.accessKey** | Defines the access key. Add the parameter to set your own **accesskey** credentials. | By default, **accessKey** is automatically generated. |
```

Comment:
`accessKey` is not `controller-manager` inline but in `minio` chart of `controller-manager`, so `Parameter` is `controller-manager.minio.accessKey `